modules/tensor_mechanics/test/tests/rom_stress_update/tests

```diff
@@ -106,6 +106,15 @@
                'cut because the computed number of substeps exceeds the maximum_number_substeps '
                'parameter.'
     []
+
+    [3tile]
+      type = 'CSVDiff'
+      input = '3tile.i'
+      csvdiff = '3tile_out.csv'
+      detail = 'where 3 tiles overlap.'
```

Comment:
I think you're doing more than just 3 tile overlapping right? Maybe something like `for a variety of different tile overlapping conditions`